tests/unit/core/i18n/test_utils.js

```diff
@@ -475,5 +486,25 @@ describe('i18n utils', () => {
       sinon.assert.calledWith(toLocaleStringSpy, 'fr');
       sinon.assert.notCalled(numberFormatSpy);
     });
+
+    describe('addonType()', () => {
+      it('translates internal addon types', () => {
+        const i18n = utils.makeI18n({}, 'en', FakeJed);
+        expect(i18n.addonType(ADDON_TYPE_DICT)).toEqual('dictionary');
+        expect(i18n.addonType(ADDON_TYPE_EXTENSION)).toEqual('extension');
+        expect(i18n.addonType(ADDON_TYPE_LANG)).toEqual('language pack');
+        expect(i18n.addonType(ADDON_TYPE_OPENSEARCH)).toEqual('search plugin');
+        expect(i18n.addonType(ADDON_TYPE_THEME)).toEqual('theme');
+        // this one seems deprecated
```

Comment:
It is but it will still appear for older (ESR) clients I think? I would include it as "theme" as well.